src/components/Grid/GridComponents.js

```diff
@@ -47,7 +47,7 @@ const Col = ({ className = '', cols = -1, style, children, content = 'expand', .
   )
 }
 Col.propTypes = {
-  children: PropTypes.node.isRequired,
+  children: PropTypes.node,
```

Comment:
Is there any reason to removing `isRequired`? Because as I understand this component, this is container for other components, so it should get some children.

Reply:
The reason is if you want a `Col` to take up space but don't want to put an empty tag inside...
````js
  <Grid>
    <Row>
      <Col cols={4} />
      <Col cols={8}>
        ...content that will render on the right most 8 columns....
      </Col>
    </Row>
  <Grid>
````

Reply:
Hmmm... true, but maybe in this case better use similar props like in patternfly-react: http://patternfly-react.surge.sh/patternfly-3/index.html?knob-fluid=true&selectedKind=patternfly-react%2FLayouts%2FGrid&selectedStory=Grid&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybooks%2Fstorybook-addon-knobs

In this case your example will look like this:

```jsx
<Grid>
    <Row>
      <Col xsOffset={4} cols={8}>
        ...content that will render on the right most 8 columns....
      </Col>
    </Row>
  <Grid>
```

Reply:
Ok I can see that.  We don't use that approach anywhere currently, but I've added it regardless.